Challo/Challo/Common/Services/ImageService/ImageService.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ImageService.swift
+//  Challo
+//
+//  Created by Shao Yi on 10/4/21.
+//
+
+import SwiftUI
+import ImageKitIO
+import SDWebImageSwiftUI
+
+struct ImageService: ImageServiceProtocol {    
+        
+    private static func generateFullUrl(path: String, height: CGFloat, width: CGFloat) -> String {
+        ImageKit.shared.url(path: path, transformationPosition: .QUERY)
+            .height(height: Int(height))
+            .aspectRatio(width: Int(width), height: Int(height))
+            .create()
+    }
+    
+    static func loadImage(path: String, height: CGFloat = 400, width: CGFloat = 400,
+                          onSuccess: ((UIImage, Data?) -> Void)? = nil) -> Image {
+        let fullUrl = generateFullUrl(path: path, height: height, width: width)
+        
+        let image = WebImage(url: URL(string: fullUrl))
+            .onSuccess { image, data, _ in
+                onSuccess?(image, data)
+            }
+            .resizable()
+            .indicator(.activity)
+            .transition(.fade(duration: 0.5))
+//            .placeholder {
+//                Rectangle().foregroundColor(.gray)
```

Comment:
Is this still needed? If not can remove instead of commenting out

Reply:
Oops forgot about that